Content.Tests/DMProject/Tests/Tree/Const/const_fn_badarg.dm

```diff
@@ -0,0 +1,9 @@
+// COMPILE ERROR
+
+/proc/somered()
+	return 127
+
+var/const/reddish = rgb(somered(),0,255) // error: constant initializer required
+
+/proc/RunTest()
+	var/const/reddish = rgb(somered(),0,0) // error: constant initializer required
```

Comment:
Only one of these erroring would lead the test to pass, which could hide an issue if the other one ends up compiling fine.